examples/fei2_classical.jl

```diff
@@ -324,18 +325,17 @@ fig#hide
 
 fig = Figure()#hide
 ωidx = 60
-target_ω = ωs(sc)[ωidx]
+target_ω = ωs[ωidx]
 
 ## Binning
 ax_left = Axis(fig[1,2],title="Δω=0.3 meV (Binned)", aspect=true)
 
-params = unit_resolution_binning_parameters(sc)
-params.binstart[1:2] .= -3
-params.binend[1:2] .= 3
-
-params.binstart[4] = target_ω
-params.binend[4] = target_ω + 0.15
-params.binwidth[4] = 0.3
+params = BinningParameters(
+    [-3,-3, 0, target_ω],           # Bin start (h,k,l,ω)
+    [3, 3, 0.15, target_ω + 0.15],  # Bin end
+    [.2, .2, .3, 0.3]               # Bin width
+)
+Sunny.bin_rlu_as_absolute_units!(params, cryst.recipvecs)
```

Comment:
It seems to me that this should be called `bin_absolute_units_as_rlu!`. The bins are specified in absolute units (used default orthogonal coordinates), and I'm trying to convert them to RLU. Let me know if I am confused.

Reply:
Ugh, yeah, I see how absolute is a lot nicer here. Should we do a short call to brainstorm the ideal user interface?

Reply:
Given that `bin_rlu_as_absolute_units!` is for now private, I think we can merge this PR and fix up later.

Reply:
(Oh and I should say, using the `unit_resolution` stuff makes the x and y axes as high resolution as possible given the lattice size at essentially zero additional computational cost)

Reply:
Thanks for the tips. I suspected I was abusing the functionality. My goal was just to get the figure to look like this,

<img width="777" alt="image" src="https://github.com/SunnySuite/Sunny.jl/assets/34151623/c1086e84-4530-4c1d-abfd-a2fc1b0ce2be">

i.e., to remove the skew. In other words, I wanted to define regular bins in absolute units directly. I get this is non-ideal from a certain perspective, but I think this is the way one would want to visualize these data.

Your second approach will work, but if I don't fudge the bin widths, there will be gaps. So, for example, the following gets the desired effect.
```
params = unit_resolution_binning_parameters(sc)
params.covectors[1:3,1:3] .= cryst.recipvecs
omega_width = 0.3
params.binstart[4] = target_ω
params.binend[4] = target_ω + (sc.Δω/2)
params.binwidth[4] = omega_width
params.binwidth[1:2] .*= 2

params.binstart[1], params.binstart[2] = -3, -3
params.binend[1], params.binend[2] = 3, 3

integrate_axes!(params, axes = [3,4])
```

<img width="779" alt="image" src="https://github.com/SunnySuite/Sunny.jl/assets/34151623/e06d8e62-1043-4b31-9486-58b6ab2725c8">

Reply:
Oh darn yeah I see. The root cause is the eigenvalues of recipvecs are not all one or whatever. Maybe try multiplying the binwidth by `1.64` instead? (Close to largest eigenvalue of recipvecs)

Reply:
Apart from the questions about `unit_resolution_binning_parameters`, this looks great to me! You have my thumbs up after convergence with @Lazersmoke re. binning interface questions.

Reply:
I find the minimal number necessary for rescaling the binwidths (to avoid empty bins) is about 1.78. It might be aesthetically nicer with a lower resolution, but I'll leave it at that. This can always be changed later.